client/lib/analytics/ad-tracking/record-order.js

```diff
@@ -533,6 +536,25 @@ function recordOrderInJetpackGA( cart, orderId, wpcomJetpackCartInfo ) {
 	}
 }
 
+/**
+ * Records an order in the WordPress.com GA4 Property
+ *
+ * @param {object} cart - cart as `ResponseCart` object
+ * @param {number} orderId - the order id
+ * @param {object} wpcomJetpackCartInfo - info about WPCOM and Jetpack in the cart
+ * @returns {void}
+ */
+function recordOrderInWPcomGA4( cart, orderId, wpcomJetpackCartInfo ) {
+	if ( wpcomJetpackCartInfo.containsWpcomProducts ) {
+		const sendToJetpack = false;
+		fireEcommercePurchaseGA4(
```

Comment:
This function is named `fireEcommercePurchaseGA4()` - is it intended to track only ecommerce plan purchases? However, the function in which it is housed implies that we track all purchases/orders in WPCOM. So which one of these is applicable?

Reply:
Actually, reading it like that -- the naming is a bit confusing! It's because this event takes in the cart, and allows us to track with the Ecommerce views in Google Analytics. Those take in a cart, price, etc. So it's not just for ecommerce sales, but for any sales (that allow us to use the ecommerce views/reports).